### PR TITLE
Fixes #28195 - publish yum repo content view without filters

### PR DESCRIPTION
**Pull request by ianballou**
This PR allows for content views to be published with Pulp 3 Yum repos.  This is WIP until the tests check out.

### CODE REVIEW

**Comment by ianballou** [comment]
The last failing test is a VCR one that I need to re-record in a pulpcore 3.0 environment.  I realize I was on master when I first recorded it.

File: app/lib/actions/katello/repository/clone_to_environment.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@ def plan(repository, environment)
               plan_action(Repository::Create, clone, true, false)
             else
               #only clear if it should be empty, but its not
-              plan_optional_pulp_action([Actions::Pulp::Repository::Clear], clone, SmartProxy.pulp_master) if (!clone.yum? || !clone.empty_in_pulp?)
+              plan_optional_pulp_action([Actions::Pulp::Repository::Clear], clone, SmartProxy.pulp_master) unless SmartProxy.pulp_master.pulp3_support?(clone)
```
Comment (jlsherrill) [inline app/lib/actions/katello/repository/clone_to_environment.rb]: plan_optional_pulp_action should ignore it if its pulp3, so i don't think we need that unless check there

Comment (ianballou) [reply app/lib/actions/katello/repository/clone_to_environment.rb]: right, missed that.

**Comment by ianballou** [comment]
@jlsherrill should be all set here now.

**Comment by ianballou** [comment]
[test katello]

**Comment by ianballou** [comment]
[test katello]

**Pull request merged**